xml/admin_monitoring_alerting.xml

```diff
@@ -1045,14 +318,10 @@ ALERTS{alertname="<replaceable>ALERT_NAME</replaceable>", alertstate="pending|fi
    <para>
     If you want to get notified about &prometheus; alerts via SNMP traps, then
     you can install the &prometheus; &alertmanager; SNMP trap receiver via
-    &deepsea;. To do so you need to enable it in the Pillar under the
+    &cephadm;. To do so you need to enable it in the Pillar under the
```

Comment:
Pillar will need to go here as well, though there's no replacement documentation in the moment as the SNMP trap receiver feature is under development.

Reply:
Currently commented out :) we'll leave until we have SNMP docs